engine/table/src/main/java/io/deephaven/engine/util/jpy/JpyInit.java

```diff
@@ -53,15 +53,17 @@ private static void init(JpyConfigExt jpyConfig) {
         jpyConfig.initPython();
         jpyConfig.startPython();
         log.info().append("Started Python interpreter").endl();
-        markJvmReady();
+        mark_ready_and_check_env();
     }
 
-    private static void markJvmReady() {
+    private static void mark_ready_and_check_env() {
         // noinspection EmptyTryBlock,unused
         try (
                 final PyModule deephavenJpyModule = PyModule.importModule("deephaven_internal.jvm");
-                final PyObject obj = deephavenJpyModule.callMethod("ready")) {
+                final PyObject readyObj = deephavenJpyModule.callMethod("ready");
+                final PyObject checkObj = deephavenJpyModule.callMethod("check_py_env")) {
```

Comment:
I'm not worrying about this too much; but what I meant was we can have `deephaven_internal.jvm.mark_ready_and_check_env` to save us from an additional `callMethod` round trip. On the other hand, I doubt it makes any sort of practical difference, and it keeps the python side a bit "cleaner".

Reply:
yeah, I wanted to be able to reuse the check_py_env() for the embedded case.